src/forge/actors/reference_model.py

```diff
@@ -115,7 +87,11 @@ def __post_init__(self):
     @endpoint
     async def setup(self):
         engine_config = {f.name: getattr(self, f.name) for f in fields(self)}
-        self.engine = ForgeEngine(ForgeJobConfig(**engine_config))
+        engine_config = ForgeJobConfig(**engine_config)
+        engine_config.checkpoint.folder = (
+            ""  # hardcode to empty to force load from initial_load_path
```

Comment:
Where does ref model get the initial weights after the change?

Reply:
Can you add a todo to make this configurable in the future? Sometimes you do want to update your reference model just very infrequently.

Reply:
@JenniferWang that would have to be on the titan side right? Also, I think this module will be reusable for any "forward pass" model in the near future and not just for reference model. So I don't know if it's worth doing anything more than this now.

Reply:
can this not just be accomplished by setting this in the config?

```
  checkpoint:
    enable: true
    initial_load_path: hf://${model}
    folder: "" 
```